src/Group_User.php

```diff
@@ -612,7 +612,12 @@ public static function showForGroup(Group $group)
                 echo "\n<tr class='tab_bg_" . ($user->isDeleted() ? '1_2' : '1') . "'>";
                 if ($canedit) {
                     echo "<td width='10'>";
-                    Html::showMassiveActionCheckBox(__CLASS__, $data["linkid"]);
+                    if (
+                        Session::canViewAllEntities()
+                        || Session::haveAccessToOneOfEntities(Profile_User::getUserEntities($data["id"], false))
+                    ) {
```

Comment:
Using the `can` method will do the same checks, by checking the result of `User::canUpdateItem()` and will also handle any specific plugin override.
```suggestion
                    if ($user->can($user->fields['id'], UPDATE)) {
```